### DISCUSSION TITLE
Icons missing from profile page

### DISCUSSION THREAD

**Discussion by ThaliaMyers** [elrumo/macOS_Big_Sur_icons_replacements#1350] [Help with the website]
All of the icons I have uploaded are now missing from my uploaded icons page, but when I search for the icon by its name it appears in the search results. Is this a problem on my end or is it a bug with the website?

![Screen Shot 2022-01-05 at 9 59 39 PM (2)](https://user-images.githubusercontent.com/79024214/148326816-bbe1c19d-68b6-45c3-b164-b742da9b83c9.png)
![Screen Shot 2022-01-05 at 9 59 59 PM (2)](https://user-images.githubusercontent.com/79024214/148326818-30fb4c41-722e-4d8f-9b20-2f853ff98741.png)

**Comment by AlephCubed** [comment]
I have the same issue. Only 2 icons appear on my profile page even though all of them are still on the website.

**Comment by orignic** [comment]
Mine have all been deleted from my profile and from the website as a whole. Which is fine with me if there's a reason for it, but I was given no reason and can't find any information about why it may have happened.
